### PR TITLE
Ui updates

### PR DESCRIPTION
**Pull request by NWeber-SL**
Three new articles:
  * inspect/filter-results.md
  * inspect/tag-build.md
  * inspect/view-results.md

### CODE REVIEW

File: SUMMARY.md [hunk 1]
```diff
@@ -64,21 +64,21 @@
        * [Tomcat Integration](using-inspect-protect/protect/protect-java/tomcat-integration.md)
        * [Websphere Configuration](using-inspect-protect/protect/protect-java/websphere-configuration.md)
        * [Wildfly Configuration](using-inspect-protect/protect/protect-java/wildfly-configuration.md)
-  * Integrating ShiftLeft with Your CI/CD System
-    * [Integrating with Jenkins Builds](using-inspect-protect/integrating-with-shiftleft/integrating-jenkins-builds/integrating-jenkins-builds.md)
+   * Using the ShiftLeft Dashboard
+       * [The Vulnerabilities Dashboard](using-inspect-protect/using-dashboard/vulnerability-dashboard.md)
+  * Integrating with Your Build and CI/CD Systems
+    * [Integrating with Jira](using-inspect-protect/integrating-with-shiftleft/jira-integration.md)
```
Comment (vlad-alexandru-ionescu) [inline SUMMARY.md]: Jira is not a CI/CD system. It should be classified differently. Or alternatively, we can rename `Integrating with Your Build and CI/CD Systems` to `Workflow Integrations` to accommodate for broader topics.

File: using-inspect-protect/inspect/filter-results.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Filtering Analysis Results
```
Comment (vlad-alexandru-ionescu) [inline using-inspect-protect/inspect/filter-results.md]: Did you mean to link this page in the ToC as well?

File: using-inspect-protect/inspect/tag-build.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Tagging a Build Based on Analysis Results
```
Comment (vlad-alexandru-ionescu) [inline using-inspect-protect/inspect/tag-build.md]: I think it's a misnomer.

The purpose of this is to tag the build based on branch name - not on the results. It doesn't depend on what the results are - the analysis will still be tagged this way.

Comment (vlad-alexandru-ionescu) [reply using-inspect-protect/inspect/tag-build.md]: This page is mixing tagging and build rules. These should be separate concepts.

File: using-inspect-protect/inspect/view-results.md [hunk 4]
```diff
@@ -2,27 +2,43 @@
...
 * [Identifying and Getting Additional Information on Top OWASP and SANS Vulnerabilities](#identifying-and-getting-additional-information-on-top-owasp-and-sans-vulnerabilities)
+* [Analysis Results Notification](#analysis-results-notification)
 
-## Analysis Results Notification
-
-When you submit an application with a new library, it may take ShiftLeft Inspect additional time to complete a full analysis. In those situations, ShiftLeft Inspect initially displays the partial analysis results, which is indicated by a message on the application's card and in the Vulnerabilities Dashboard. 
+## Displaying Results by Application Version
 
- ![Partial Results Message](img/partial-results.png)
+If you use ShiftLeft to analyze and secure multiple versions of an application (for example production, staging, test and  UAT), you can view results for only a particular version. To do so,
```
Comment (vlad-alexandru-ionescu) [inline using-inspect-protect/inspect/view-results.md]: We shouldn't recommend this - the build tagging is only meant for branches. Not for prod / staging / test differentiation.

File: SUMMARY.md [hunk 5]
```diff
@@ -64,21 +64,21 @@
...
       * [Verify Jenkins Integration](using-inspect-protect/integrating-with-shiftleft/integrating-jenkins-builds/verify-jenkins-integration.md)
-    * [Integrating with Bamboo Builds](using-inspect-protect/integrating-with-shiftleft/integrating-bamboo-builds.md)
+    * [Integrating with Bamboo](using-inspect-protect/integrating-with-shiftleft/integrating-bamboo-builds.md)
     * [Integrating with Docker](using-inspect-protect/integrating-with-shiftleft/integrating-docker.md)
-    * [Integrating with GoCD Builds](using-inspect-protect/integrating-with-shiftleft/integrating-gocd-builds.md)
-    * [Integrating with TeamCity Builds](using-inspect-protect/integrating-with-shiftleft/integrating-teamcity-builds.md)
-    * [Integrating with Travis Builds](using-inspect-protect/integrating-with-shiftleft/integrating-travis-builds.md)
+    * [Integrating with GoCD](using-inspect-protect/integrating-with-shiftleft/integrating-gocd-builds.md)
+    * [Integrating with TeamCity](using-inspect-protect/integrating-with-shiftleft/integrating-teamcity-builds.md)
+    * [Integrating with Travis](using-inspect-protect/integrating-with-shiftleft/integrating-travis-builds.md)
```
Comment (vlad-alexandru-ionescu) [inline SUMMARY.md]: Wouldn't these changes cause broken links? Do we really need to remove "Builds" in here?

File: SUMMARY.md (hunk 1 above)
Comment (NWeber-SL) [reply SUMMARY.md]: I like that. Done.

File: using-inspect-protect/inspect/filter-results.md (hunk 2 above)
Comment (NWeber-SL) [reply using-inspect-protect/inspect/filter-results.md]: Will do after I move it to new place under Using the ShiftLeft Dashboard

File: SUMMARY.md (hunk 5 above)
Comment (NWeber-SL) [reply SUMMARY.md]: No the title of the article is changed, but NOT the file name. So links won't break

File: using-inspect-protect/inspect/fail-build.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+1. [Create a filter that defines the fail criteria](../using-dashboard/filter-results.md#creating-a-filter).
+2. [Save the filter](../using-dashboard/filter-results.md#saving-a-filter).
+3. In the Save Filter dialog, check **Build Rule for Org**.
+
+     ![Save Build Rule](img/build-rule.jpg)
+
+4. Use the following bash script, which invokes ShiftLeft's public API. 
+   The script shows 1 for success if the ShiftLeft Inspect results don’t have any vulnerabilities that match the build rule criteria. The script shows 0 for failure, if the build has vulnerabilities that match the build rule criteria.
+ 
+```
```
Comment (vlad-alexandru-ionescu) [inline using-inspect-protect/inspect/fail-build.md]:
```suggestion
```bash
```

**Pull request merged**